src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java

```diff
@@ -32,6 +33,12 @@
     public static void main(String[] args) throws Exception {
 
         try (CumulusServer server = new CumulusServer(Configuration.getCumulusConf())) {
+            Properties exportpro = new Properties();
+
+            exportpro.load(CumulusExport.class.getClassLoader().getResourceAsStream("cumulusExport.properties"));
+            boolean limited = Boolean.parseBoolean(exportpro.getProperty("limited"));
+            int counter = Integer.parseInt(exportpro.getProperty("counter"));
+
```

Comment:
Property loading and general handling should be part of the already existing `Configuration.java`.

Reply:
I think a property file is easier to work with than a yml file.